src/amo/pages/Addon/styles.scss

```diff
@@ -115,19 +114,13 @@
 }
 
 .Addon .AMInstallButton {
+  width: 100%;
+
   @include respond-to(medium) {
     @include margin-start(auto);
 
     align-content: center;
     align-self: center;
-  }
-}
-
-.Addon .AMInstallButton-button,
```

Comment:
do we still need this? the button size looks different from what I see on dev (on ~ mobile)

Reply:
I checked my local page vs. dev for both the install button and the Get Firefox button and they look identical to me. What difference are you seeing? Can you share a screenshot?

Reply:
To be clear, I only see this with the install version (the Firefox button looks fine) 

So this is what I see this on Dev:

![Alt text](https://monosnap.com/image/rVQQXH5IIBT9yHem2Eht5YNH4USQ32.png)

and this what I see locally:

![Alt text](https://monosnap.com/image/L4yotwzVxqvsSc2bJ6tb27TA6dPbBX.png)

Reply:
Yep, it should be full width on small screens.